Workflow API docs (#2915)
dapr/docs: daprdocs/content/en/concepts/building-blocks-concept.md

| [**Secrets**]({{< ref "secrets-overview.md" >}}) | `/v1.0/secrets` | Dapr provides a secrets building block API and integrates with secret stores such as public cloud stores, local stores and Kubernetes to store the secrets. Services can call the secrets API to retrieve secrets, for example to get a connection string to a database.
| [**Configuration**]({{< ref "configuration-api-overview.md" >}}) | `/v1.0-alpha1/configuration` | The Configuration API enables you to retrieve and subscribe to application configuration items for supported configuration stores. This enables an application to retrieve specific configuration information, for example, at start up or when configuration changes are made in the store. 
| [**Distributed lock**]({{< ref "distributed-lock-api-overview.md" >}}) | `/v1.0-alpha1/lock` | The distributed lock API enables you to take a lock on a resource so that multiple instances of an application can access the resource without conflicts and provide consistency guarantees.  

> | [**Workflows**]({{< ref "workflow-overview.md" >}}) | `/v1.0-alpha1/workflow` | The workflow API enables you to define a processes or data flows that span multiple microservices via an embedded workflow engine. With this built-in workflow engine, you can easily integrate with existing Dapr building blocks while maintaining portability.

Reviewer: ```suggestion
| [**Workflows**]({{< ref "workflow-overview.md" >}}) | `/v1.0-alpha1/workflow` | The Workflow API enables you to define processes or data flows that span multiple microservices via the dapr-embedded workflow engine or supported external workflow components. With the Workflow API, you can easily integrate with existing Dapr building blocks while maintaining portability.
```

> | [**Workflows**]({{< ref "workflow-overview.md" >}}) | `/v1.0-alpha1/workflow` | The Workflow API enables you to define processes or data flows that span multiple microservices via the Dapr-embedded workflow engine or supported external workflow components. With the Workflow API, you can easily integrate with existing Dapr building blocks while maintaining portability.

Reviewer 2: ```suggestion
| [**Workflows**]({{< ref "workflow-overview.md" >}}) | `/v1.0-alpha1/workflow` | The Workflow API enables you to define long running, persistent processes or data flows that span multiple microservices using Dapr workflows or workflow components. The Workflow API can be combined  with other Dapr API building blocks, for example a workflow can call another service with service invocation, or retrieve secrets, providing flexibility and portability. 
```
